.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "ros.distro": "humble"
+}
```

Comment:
Seems like this file was added accidentally. Please remove.

Reply:
Removed